src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep.java

```diff
@@ -147,7 +147,7 @@ public String getInheritFrom() {
 
     @DataBoundSetter
     public void setInheritFrom(@CheckForNull String inheritFrom) {
-        if (inheritFrom != null && inheritFrom.equals("-")) { // Added to make KubernetesPipelineTest.runInPodNestedExplicitInherit() pass
+        if (inheritFrom != null && inheritFrom.equals(",default,")) { // Added to make KubernetesPipelineTest.runInPodNestedExplicitInherit() pass
```

Comment:
Why this strange string? The commit comment says something about escaping in XML and Java but I do not see how that would be relevant. The value will only ever be used inside the snippet generator.